concepts/framework/flow-concept.md

```diff
@@ -43,8 +43,35 @@ In Shopware, you have multiple interfaces and classes for different types of eve
 
 Once the action on the Storefront or from the app happens, the FlowDispatcher will dispatch FlowEventAware to the FlowExecutor. From here, the FlowExecutor will check the condition to decide whether to execute the action.
 
-![Flow builder concept for flow sequence](../../assets/flow-concept-1.png)
+```mermaid
```

Comment:
GitHub has difficulties displaying the rich diff here (i.e. rendering the mermaid). We should check after the deployment if it is rendered correctly.